features/FEATURE_LWIP/lwip-interface/lwip_stack.c

```diff
@@ -1147,9 +1147,11 @@ static nsapi_error_t mbed_lwip_socket_listen(nsapi_stack_t *stack, nsapi_socket_
 {
     struct lwip_socket *s = (struct lwip_socket *)handle;
 
+#if LWIP_TCP
```

Comment:
To further reduce code size, I reckon the entirety of both these functions probably should be wrapped in the `#if` so they just return "unsupported" if !TCP.

Reply:
Updated flagging.